src/Components/Links.js

```diff
@@ -13,6 +13,13 @@ function Links({ links }) {
     instagram: '#E4405F',
   }
 
+  const bgcolors = {
+    youtube: 'rgb(255, 55, 55)',
```

Comment:
this might be easier to read with a single object

```js
const socials = {
   YouTube: {
      color: 'FF0000',
      background: 'rgb(255, 55, 55)' // would be better in hex to be consistent
   }
}
```